site-templates/styles/components/navigation.css

```diff
@@ -45,5 +44,4 @@
   bottom: 0;
   left: 0;
   width: 3px;
-  background-color: var(--brand-color);
```

Comment:
These seem like they would change the look of our current layout.

Are they just being overwritten already by some other class?

Reply:
This is for the openapi.josn files in the left navigation, they are the only ones different to the rest of the items in navigation.